eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/UtxoSpec.hs

```diff
@@ -64,6 +67,49 @@ spec =
         [fromNativeScript spendingScript, fromNativeScript spendingScript]
           ++ extraScripts
           ++ extraScripts
+
+    it "deregistering returns the deposit" $ do
+      let
+        keyDeposit = Coin 2
+        -- This is paid out as the reward
+        govActionDeposit = Coin 3
+      modifyPParams $ \pp ->
+        pp
+          & ppKeyDepositL .~ keyDeposit
+          & ppGovActionDepositL .~ govActionDeposit
+      stakeCred <- KeyHashObj <$> freshKeyHash
+      rewardAccount <- registerStakeCredential stakeCred
+      otherStakeCred <- KeyHashObj <$> freshKeyHash
+      otherRewardAccount <- registerStakeCredential otherStakeCred
+      khStakePool <- freshKeyHash
+      registerPool khStakePool
+      submitTx_ . mkBasicTx $
+        mkBasicTxBody
+          & certsTxBodyL
+            .~ SSeq.fromList
+              [ DelegTxCert stakeCred (DelegStakeVote khStakePool DRepAlwaysAbstain)
+              , DelegTxCert otherStakeCred (DelegStakeVote khStakePool DRepAlwaysAbstain)
+              ]
+      _ <- tryLookupReward stakeCred
```

Comment:
What is this line testing? Why not check the expected value?

Reply:
Removed it, there are assertions for the reward addresses on the next two lines anyways